pkg/reflector/reflector.go

```diff
@@ -280,6 +294,53 @@ func ReflectorFromConfig(config ReflectorConfig) (*Reflector, error) {
 	}, nil
 }
 
+func emitMetricFromFile() error {
```

Comment:
ideally we should have a test for this

Reply:
interesting, I thought about it but wasn't sure if there is existing solution to mock Datadog agent in order to fully test this function, I am going to search a bit and circle back unless you already know there is such a thing

Reply:
I did search around and found the stats lib got a statstest handler that I could use to mock the agent: https://github.com/segmentio/stats/blob/main/statstest/handler.go, but I played around with it and couldn't make it work. I am now leaning toward not chasing to get it done perfectly as I think I need to grok the concept a bit more to fully understand it.

Reply:
for reference, this test also mocks stats package: https://github.com/segmentio/ctlstore/blob/master/pkg/globalstats/stats_test.go. But what you have now is fine to me